.github/workflows/release_mac.yml

```diff
@@ -0,0 +1,70 @@
+name: MacRelease
+
+on:
+  push:
+    branches:
+      - rel-*
```

Comment:
let's also run this for pushes on master. This way these pipelines will be run frequently